updatecli/scripts/run-make-generate.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -eux
+
+mkdir -p build/data
+
+make download
+
+make generate
+
+git diff
```

Comment:
This seems to be essentially the same thing as run-go-generate: 
https://github.com/k3s-io/k3s/blob/83a3e851d3d809c12b24789e28746c0ce4620359/updatecli/scripts/run-go-generate.sh#L5-L7

Is there any particular reason to create a new different version of those same steps?